src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt

```diff
@@ -24,7 +24,7 @@ class RsFileStub : PsiFileStubImpl<RsFile> {
 
     object Type : IStubFileElementType<RsFileStub>(RsLanguage) {
         // Bump this number if Stub structure changes
-        override fun getStubVersion(): Int = 69
+        override fun getStubVersion(): Int = 71
```

Comment:
Yeah, an arbitrary increment is fine here, we probably won't run out of Ints in the nearest future :)